app/js/enhancedDate.js

```diff
@@ -1,5 +1,144 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+/*
+ * Carolyn Velez
+ * JSCRIPT 300
+ * 04/14/15
+ */
 
+var enhancedDate = (function iife() {
+
+	// private
+	var myDate;
+
+	// public API
+	return {
+
+		setDate: function setDate(specificDate) {
+			if (specificDate === undefined) {
+				myDate = new Date;
+			} else {
+				myDate = specificDate;
+			}
+		},
+
+		getDate: function getDate(dateInMilliseconds) {
```

Comment:
If myDate hasn't been set to a Date instance yet, you'll want to do that first so that you don't return undefined if it wasn't.